src/lib/AnnotationControls.ts

```diff
@@ -84,21 +84,53 @@ export default class AnnotationControls {
     private handleFullscreenExit = (): void => this.handleFullscreenChange(false);
 
     /**
-     * Region comment button click handler
+     * Deactive current control button
      */
-    private handleRegionClick = (onRegionClick: RegionHandler) => (event: MouseEvent): void => {
+    public deactivateCurrentControl = (): void => {
+        if (!this.currentActiveControl) {
+            return;
+        }
+
+        switch (this.currentActiveControl) {
+            case ANNOTATION_MODE.region:
```

Comment:
It seems like we're planning ahead for more buttons, which is great. Maybe we could create a map for mode type -> method, like so?

```
this.controlsMap = {
    [ANNOTATION_MODE.region]: this.updateRegionButton
}
```